style.css

```diff
@@ -0,0 +1,9 @@
+* {
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Comment:
Sind das unterschiedliche Schriften für unterschiedliche OS?

Reply:
das sind fallbacks falls deine Browser oder dein Endgerät eine Schriftart nicht installiert hat. 
Das war auch in VS Code schon hinterlegt.